src/Savegame/Soldier.cpp

```diff
@@ -115,6 +117,7 @@ void Soldier::load(const YAML::Node& node, const Mod *mod, SavedGame *save, cons
 {
 	_id = node["id"].as<int>(_id);
 	_name = node["name"].as<std::string>();
+	_callsign = node["callsign"].as<std::string>(generateCallsign(_rules->getNames()));
```

Comment:
default empty (same as for _name)

Reply:
This was added because I wanted to enable player's who are loading a pre-existing save to have new callsigns generated for their soldier. Should I move that elsewhere or how to go about that?

Reply:
Honestly, I would ignore that. Just start a new campaign. We do a "migration" sometimes, but only if it cannot be avoided.